samples/drivers/eeprom/boards/gd32f450i_eval.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022 BrainCo Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		eeprom-0 = &eeprom0;
+	};
+};
```

Comment:
nit: should be part of the board itself, as do e.g. `led0`

Reply:
OK, move it to board level dts.

Reply:
done.